src/test/java/org/codehaus/mojo/sql/SqlExecMojoTest.java

```diff
@@ -28,106 +30,86 @@
 import org.apache.maven.settings.Server;
 import org.apache.maven.settings.Settings;
 import org.apache.maven.shared.filtering.MavenFileFilter;
-import org.junit.FixMethodOrder;
-import org.junit.runners.MethodSorters;
+import org.codehaus.plexus.component.repository.exception.ComponentLookupException;
 import org.sonatype.plexus.components.sec.dispatcher.SecDispatcher;
 
 /**
- * Unit test for simple SqlExecMojo.<br/>
- * ATTENTION: These are no unit tests, cause the are order dependent.<br/>
- * This must be fixed. At the moment it is handled having appropriate method names.
```

Comment:
so we no longer depend on test method orders? how that was solved?

Reply:
Now each test creates own instance of mojo (using createMojo() method) and also uses own data (tables) to tests.

The tests are run using single db instance. There were multiple tests that assumed that person table existed or didn't exists. So I added own test data (sql files) to the tests so they do not use same person table. Now the tests have their own tables.

I also tried to reset database after each test, but didn't find proper way to do it using Axios.
Also tried to change Axion to HSQLDB + reset data but then some login tests failed.

Reply:
There were some issues.. I changed Axion to HSQLDB which is active project. Now each test is using own database (UUID is added to the database name). This way the tests should now be totally independent.

The HSQLDB change and test fixes are not directly related to the PR 26. I can make these changes to other PR with one commit and create new branch for the PR 26.